main/options/options.cc

```diff
@@ -173,7 +173,11 @@ UnorderedMap<string, core::StrictLevel> extractStricnessOverrides(string fileNam
                         case YAML::NodeType::Sequence:
                             for (const auto &file : child.second) {
                                 if (file.IsScalar()) {
-                                    result[file.as<string>()] = level;
+                                    string key = file.as<string>();
+                                    if (key.find("/") != 0 && key.find("./") != 0) {
```

Comment:
Given that this logic appears in two places, I considered putting it into a helper function maybe somewhere in `FileOps`, but I also am afraid that by having a helper function a future programmer—including possibly me—would assume that it implements file path normalization more fully than it does.